src/components/Nav.scss

```diff
@@ -0,0 +1,33 @@
+@import "../colors";
+
+.navbar {
+  background-color: $lightGrey;
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  margin-bottom: 5rem;
+
+}
+
+.nav__logo {
+  display: flex;
+  color: $darkPurple;
+  align-items: center;
+
+  &--icon {
+    margin-right: 2rem;
```

Comment:
Maybe we can use the sass variables from the `../spacing.scss` file to keep everything consistent over time